Client/src/main/java/org/projectcardboard/client/view/battleview/MapBox.java

```diff
@@ -214,13 +221,14 @@ private void clickCell(int row, int column) {
           if (GameController.getInstance().getAvailableActions().canDeployMinionOnSquare(gameMap,
               player, card, row, column)) {
             battleScene.getController().insert(card, row, column);
-            System.out.println("Insert " + battleScene.getHandBox().getSelectedCard().getCardId());
+            // System.out.println("Insert " +
```

Comment:
Should remove this comment and the others. Maybe they should be switched to `.debug` messages instead though?